src/stream/futures_unordered.rs

```diff
@@ -427,6 +436,37 @@ impl<T> Drop for FuturesUnordered<T> {
     }
 }
 
+#[derive(Debug)]
+/// Mutable iterator over all futures in the unordered set.
+pub struct FuturesUnorderedIterMut<'a, F: 'a> {
```

Comment:
I would probably just name this `IterMut` as it already is in the `futures_unordered` module.